app/mailers/devise_mailer.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+class DeviseMailer < ActionMailer::Base
+  before_action :set_mailers_url_host
+
+  DEFAULT_APP_HOST = 'lvh.me'
```

Comment:
Why are we hard-coding this instead of using `Application::Application.config.x.default_app_host`?

Reply:
refactored. noted on next time to be aware of this functionality